--- a/layout/includes/header/index.pug
+++ b/layout/includes/header/index.pug
@@ -9,8 +9,13 @@
     when 'post'
       - top_img = page.top_img || page.cover || theme.default_top_img
     when 'page'
-      - top_img = page.top_img || theme.default_top_img
+      if page.type === 'categories'
+        - top_img = returnTopImg(theme.category_img)
+      else if page.type === 'tags'
+        - top_img = returnTopImg(theme.tag_img)
+      else
+        - top_img = page.top_img || theme.default_top_img
     when 'tag'
       - top_img = theme.tag_per_img && theme.tag_per_img[page.tag] || returnTopImg(theme.tag_img)
     when 'category'
